pubspec.yaml

```diff
@@ -77,23 +81,31 @@ dependencies:
   emoji_picker_flutter: ^1.5.1
   emoji_proposal: ^0.0.1
   emojis: ^0.9.9
-  fcm_shared_isolate: ^0.1.0
+  # TODO: remove this after this PR is merged https://gitlab.com/famedly/company/frontend/libraries/native_imaging/-/merge_requests/22
+  fcm_shared_isolate:
+    git:
+      url: https://github.com/famedly/fcm_shared_isolate.git
+      ref: main
```

Comment:
- [ ] demo notification in iOS

Reply:
Any update @sherlockvn ?

Reply:
https://github.com/user-attachments/assets/735fa7ca-f25d-4d53-b14f-53a97c82e61b